src/celeritas/em/distribution/UrbanMscScatter.hh

```diff
@@ -189,7 +189,8 @@ CELER_FUNCTION auto UrbanMscScatter::operator()(Engine& rng) -> MscInteraction
     if (skip_sampling_)
     {
         // Do not sample scattering at the last or at a small step
-        return MscInteraction{true_path_, inc_direction_, {0, 0, 0}};
+        return {
+            true_path_, {0, 0, 0}, {0, 0, 0}, MscInteraction::Action::unchanged};
```

Comment:
so, the direction here is the delta_direction and will be added to inc_direction later?

Reply:
The direction is ignored if "skip sampling" is true so there's no need to copy inc_direction.